tests/test_prefect_flows.py

```
@@ -0,0 +1,103 @@
+from unittest.mock import patch, MagicMock
+
+from proxy.prefect_flows import dbtjob_v1
+
+
+def _make_task_config(**overrides):
+    config = {
+        "slug": "dbt-run",
+        "cli_profile_block": "test-block",
+        "commands": ["dbt run"],
+        "env": {},
+        "working_dir": "/tmp/working",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/working"

(S108)